src/main/java/org/sunbird/cassandra/utils/CassandraPropertyReader.java

```diff
@@ -0,0 +1,58 @@
+package org.sunbird.cassandra.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.springframework.context.annotation.PropertySource;
+
+/**
+ * This class will be used to read cassandratablecolumn properties file.
+ * @author fathima
+ *
+ */
+
+@PropertySource("/src/main/resources:cassandratablecolumn.properties")
+public class CassandraPropertyReader {
```

Comment:
Declare this as Configuration class. No need to add constructor or getInstance methods.
If you are using PropertySource - then you should be able to read it like env.getProperty(key)..